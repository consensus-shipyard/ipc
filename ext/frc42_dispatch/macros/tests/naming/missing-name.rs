use frc42_macros::method_hash;

fn main() {
	// should panic because no string or identifier provided
    let _ident_hash = method_hash!();
}
