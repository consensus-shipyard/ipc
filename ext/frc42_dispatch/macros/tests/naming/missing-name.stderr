error: unexpected end of input, expected string literal
  --> tests/naming/missing-name.rs:12:23
   |
12 |     let _ident_hash = method_hash!();
   |                       ^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `method_hash` (in Nightly builds, run with -Z macro-backtrace for more info)
